config/config_test.go

```diff
@@ -329,6 +329,10 @@ func TestReadRulesConfig(t *testing.T) {
 	c, err := getConfig([]string{"--no-validate", "--config", "../config.yaml", "--rules_config", "../rules_complete.yaml"})
 	assert.NoError(t, err)
 
+	rules, err := c.GetAllSamplerRules()
+	assert.NoError(t, err)
+	assert.Len(t, rules.UniqueFields(), 10)
+
```

Comment:
Won't this fail if any of the fields are empty?

Reply:
Do you mind to explain what do you mean by fields are empty? I'm not sure I understand what you are referring to

Reply:
It appears to me that if a field is nil, then it wont be added to the fields map. That would change the length here right?

Reply:
The field is only added if it's present in the sampler configs. Currently, the test data this test is using has ten unique fields. If someone went to change the test data, then they would need to modify this assertion to the corresponding value. So you are right, it would change the length there